### PR TITLE
Fixed component for header

### PR DESCRIPTION
**Pull request by ASGladchenko**

### CODE REVIEW

File: src/components/burger/index.tsx [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 
 import { IBurger } from './types';
 import * as Styles from './styles';
+import { positionTop } from './helpers';
 import * as Icon from '../../components/icon';
 
 const Burger = ({ data, lang, onSelectLanguage, duration = 600 }: IBurger) => {
   const burgerRef = useRef<HTMLDivElement>(null);
   const { isOpen, isAnimation, setOpen } = useDelayAnimation(duration);
+  const width = useWindowWidth();
```
Comment (cloudmenson) [inline src/components/burger/index.tsx:14]: Sort

File: src/components/burger/index.tsx [hunk 2]
```diff
@@ -18,6 +20,7 @@ const Burger = ({ data, lang, onSelectLanguage, duration = 600 }: IBurger) => {
       </Styles.BurgerIcon>
 
       <Overlay
+        top={positionTop(width)}
         isHeader
         isOpen={isOpen}
         duration={duration}
```
Comment (cloudmenson) [inline src/components/burger/index.tsx:26]: sort

File: src/components/burger/index.tsx [hunk 3]
```diff
@@ -35,7 +38,17 @@ const Burger = ({ data, lang, onSelectLanguage, duration = 600 }: IBurger) => {
           <Styles.BurgerContainer>
             {data.map((item, index) => (
               <React.Fragment key={index}>
-                <Deals text={item.text} to={item.to} heartIcon={item.icon} />
+                {item.mobile ? (
+                  <Responsive.Mobile>
+                    <Deals
+                      text={item.text}
+                      to={item.to}
+                      heartIcon={item.icon}
```
Comment (cloudmenson) [inline src/components/burger/index.tsx:46]: sort

**Review by cloudmenson — APPROVED** [review]

File: src/components/burger/helpers.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export const positionTop = (size: number) => {
+  if (size <= 767) return 0;
+
+  if (size > 767 && size <= 1366) return 85;
+
+  if (size > 1366) return 168;
+};
```
Comment (Wolshebnik) [inline src/components/burger/helpers.ts:7]: rewrite with switch
or 
last if you should delete
just return 168

**Review by Kyslytsia — APPROVED** [review]

**Review by Mish-hash — APPROVED** [review]

File: src/components/burger/helpers.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
+export const positionTop = (size: number) => {
+  switch (true) {
+    case size <= 768:
+      return 0;
+    case size > 767 && size <= 1366:
+      return 85;
+    case size > 1366:
+      return 168;
+  }
+};
```
Comment (Wolshebnik) [inline src/components/burger/helpers.ts:10]: strange switch

**Pull request merged**